tests/test_phs.py

```diff
@@ -18,6 +19,145 @@ def test_phs(get_test_url, retrieve_data):
+                rh=60,
+                v=0.1,
+                met=300 / met_to_w_m2 / a_dubois,
+                clo=0.5,
+                posture="standing",
+                wme=0,
+                model="7933-2023",
+                duration=480,
+                limit_inputs=False,
+                acclimatized=0,
+                round_output=False,
+            ),
+            dict(
+                water_loss=6345,
+                t_cr=40.8,
+                d_lim_loss_95=250,
+                d_lim_t_re=62,
+            ),
+        ),
+        (
+            dict(
+                tdb=30,
+                tr=54.2,
+                rh=35,
+                v=0.1,
+                met=300 / met_to_w_m2 / a_dubois,
+                clo=0.8,
+                posture="standing",
+                wme=0,
+                a_p=0.3,
+                f_r=0.85,
+                model="7933-2023",
+                duration=480,
+                limit_inputs=False,
+                acclimatized=0,
+                round_output=False,
+            ),
+            dict(
+                water_loss=6419,
+                t_cr=38.7,
+                d_lim_loss_95=280,
+                d_lim_t_re=149,
+            ),
+        ),
+        (
+            dict(
+                tdb=30,
+                tr=30,
+                rh=45,
+                v=1.0,
+                met=450 / met_to_w_m2 / a_dubois,
+                clo=0.5,
+                posture="standing",
+                wme=0,
+                model="7933-2023",
+                duration=480,
+                limit_inputs=False,
+                acclimatized=0,
+                round_output=False,
+            ),
+            dict(
+                water_loss=4593,
+                t_cr=38.0,
+                d_lim_loss_95=400,
+            ),
+        ),
+        (
+            dict(
+                tdb=35,
+                tr=74.6,
+                rh=30,
+                v=1.0,
+                met=250 / met_to_w_m2 / a_dubois,
+                clo=1,
+                posture="sitting",
+                wme=0,
+                a_p=0.2,
+                f_r=0.85,
+                model="7933-2023",
+                duration=480,
+                limit_inputs=False,
+                acclimatized=1,
+                round_output=False,
+            ),
+            dict(
+                water_loss=5813,
+                t_cr=37.5,
+                d_lim_loss_95=310,
+            ),
+        ),
+    ],
+)
+def test_2023_standard(inputs, expected):
+    result = phs(**inputs)
+
+    # check water_loss and core temperature
+    assert result.water_loss == pytest.approx(expected["water_loss"], rel=0.025)
+    assert result.t_cr == pytest.approx(expected["t_cr"], abs=0.3)
+
+    assert result.d_lim_loss_95 == pytest.approx(expected["d_lim_loss_95"], abs=10)
+
+    if "d_lim_t_re" in expected:
+        assert result.d_lim_t_re == pytest.approx(expected["d_lim_t_re"], abs=10)
```

Comment:
See code rabbit comment and please Temporarily mark test 3 as expected to fail until a fix can be implemented